wazo_agid/objects.py

```diff
@@ -906,33 +906,44 @@ def __init__(self, agi, cursor, context):
 class CallerID(object):
     @staticmethod
     def parse(callerid):
+        logger.debug('caller_id parse: parsing "%s"', callerid)
         m = CALLERID_MATCHER(callerid)
 
         if not m:
+            logger.debug('caller_id parse: could not match callerid, giving up')
             return
 
         calleridname = m.group(1)
         calleridnum = m.group(3)
+        logger.debug('caller_id parse: calleridname: "%s", calleridnum: "%s"', calleridname, calleridnum)
 
         if calleridname is None:
             calleridname = m.group(2)
+            logger.debug('caller_id parse: calleridname: "%s", calleridnum: "%s"', calleridname, calleridnum)
 
             if calleridnum is None and CALLERIDNUM_MATCHER(calleridname):
                 calleridnum = m.group(2)
+                logger.debug('caller_id parse: calleridname: "%s", calleridnum: "%s"', calleridname, calleridnum)
```

Comment:
do we really care to know what is the caller at each step? I mean it's annoying to have these 3 same output. We must refer to the code to know these meanings

Reply:
The point is to know exactly in which branch we go through, so yes, they are necessary. I added a small description in the last two logs.